.vscode/launch.template.json

```diff
@@ -14,7 +14,7 @@
                 "${workspaceFolder}/**/*.js",
             ],
             "env": {
-                "EXPORT_API_HOST": "https://api-ext.your-loops.com",
+                "TIDE_WHISPERER_SERVICE": "http://localhost:9127",
                 "EXPORT_HTTP_PORT": "9300",
                 // "API_HOST":"http://localhost:8009",
```

Comment:
shall we completely remove these?

Reply:
> LGTM although I added some comments and typo fix not mandatory
> I was wondering whether we should keep the "export" in the URLs? Probably yes if we don't want to break existing calls
> 
> I am able to test it locally with success

For now it is used in a batch export on some environments, it may be called from blip in the future. So I would keep it.